mantidproject/mantid
# Use boost::math::hermite

Description of work.

Hermite polynomials are available in [boost](http://www.boost.org/doc/libs/1_60_0/libs/math/doc/html/math_toolkit/sf_poly/hermite.html) and will included in the [C++17 standard](http://en.cppreference.com/w/cpp/numeric/special_math/hermite). I don't see any benefit to maintaining a separate implementation.

**To test:**



Verify removing this class is the right decision.

This is a small change with no issue number.

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
